src/get-started/codelab.md

```diff
@@ -571,9 +571,8 @@ lazily, on demand.
       }
     ```
 
- 4. In the `_RandomWordsState` class, update the `build()` method to use
-    `_buildSuggestions()`, rather than directly calling the word
-    generation library.  ([`Scaffold`][]
+ 4. We not have the `_RandomWordsState` class using a `ListView.builder`
+    rather than directly calling the word generation library.  ([`Scaffold`][]
```

Comment:
@albertodev01, I don't understand this sentence. Did you mean "We DON'T have..." But, even if you did, you have removed a sentence that provided an instruction on what to do, with a sentence that does not tell the user what to do.

Reply:
I made a typo! I meant "We now have" instead of "We not have"